app/src/main/java/com/example/masala_food_recipes/Interactor.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.masala_food_recipes
+
+import com.example.masala_food_recipes.data.DataManager
+import com.example.masala_food_recipes.data.entities.Recipe
+
+class Interactor(context: MainActivity) {
+
+    private val recipes = DataManager(context).getAllRecipesData()
+    fun underTwentyMin(): List<Recipe>{
+        val list = mutableListOf<Recipe>()
```

Comment:
I don't want the return to be Recipe item I want only recipe name, image and time to prepare for each interactor try to achieve that using groupBy and map